data/mapping/mapping.json

```diff
@@ -4,31 +4,25 @@
       "analyzer":{
         "gp_name_analyzer":{
           "type":"custom",
+          "tokenizer":"name_tokenizer",
+          "filter":[ "lowercase", "gp_title_stop" ]
+        },
+        "surgery_name_analyzer":{
+          "type":"custom",
           "tokenizer":"standard",
-          "filter":[
-            "lowercase",
-            "gp_stop"
-          ]
+          "filter":[ "lowercase" ]
+        }
+      },
+      "tokenizer": {
+        "name_tokenizer": {
+          "type": "pattern",
+          "pattern": "[^a-zA-Z']+"
         }
       },
       "filter":{
-        "gp_stop":{
+        "gp_title_stop":{
           "type":"stop",
-          "stopwords": [ 
-            "and",
-            "the",
-            "&",
-            "doctor",
-            "doctors",
-            "dr",
-            "drs",
-            "partners",
-            "practice",
-            "surgery",
-            "medical",
-            "health",
-            "centre",
-            "center" ]
+          "stopwords": [ "doctor", "doctors", "dr", "drs" ]
```

Comment:
Yup, my bad omitted `-i`. Is it worth adding `gp` and possibly `general` and `practitioner`? That would cover this result `General practitioner Dr Bernadette Duffy `